Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -80,7 +80,12 @@ protected Quaternion GoalRotation
         protected Vector3 GoalScale
         {
             get { return SolverHandler.GoalScale; }
-            set { SolverHandler.GoalScale = value; }
+            set {
+                if (SolverHandler == null)
```

Comment:
If we are going to go this path of searching fo solver handler. Can you move to util method and call for Awake, OnEnable, GoalPosition, GoalRotation, etc?

Reply:
I'm actually not sure why we needs these? Looks like they were added here: https://github.com/microsoft/MixedRealityToolkit-Unity/commit/b042ac1446aecc4d5190501620a036d868fb064d#diff-8b68ce5f0aa5c4660f69b67600085bed I'll remove them since it doesn't look like they are required. (Since we have [RequireComponent(typeof(SolverHandler))] and acquire it on Awake)

Reply:
I think we still need them on Awake